packages/scripts/package.json

```diff
@@ -44,13 +44,15 @@
 		"command-exists": "^1.2.8",
 		"cross-spawn": "^5.1.0",
 		"decompress-zip": "^0.2.2",
+		"dir-glob": "^3.0.1",
 		"eslint": "^6.1.0",
 		"jest": "^24.7.1",
 		"jest-puppeteer": "^4.3.0",
 		"js-yaml": "^3.13.1",
 		"lodash": "^4.17.15",
 		"minimist": "^1.2.0",
 		"npm-package-json-lint": "^4.0.3",
+		"prettier": "npm:wp-prettier@^1.19.1",
```

Comment:
This remind me that we need to add `engine` section and set minimum version of `npm` to 6.9.

In addition, we need to update the defaults in `check-engines` command and probably mention then now in docs given that they are quite specific:

https://github.com/WordPress/gutenberg/blob/ca0c0eb6ad8508154169b89d03c119c0a56dad0b/packages/scripts/scripts/check-engines.js#L24

https://github.com/WordPress/gutenberg/tree/master/packages/scripts#check-engines

Maybe a subsection in `Advanced Usage` on how to update `npm` to the latest version would be nice to have as well:

https://github.com/WordPress/gutenberg/tree/master/packages/scripts#advanced-usage